pkg/snapshot/overlay.go

```diff
@@ -917,35 +895,32 @@ func (o *snapshotter) magicFilePath(id string) string {
 	return filepath.Join(o.root, "snapshots", id, "fs", "overlaybd.commit")
 }
 
-func (o *snapshotter) tgtTargetConfPath(id string, key string) string {
-	name := strings.Replace(key, ":", "-", -1)
-	name = strings.Replace(name, "/", "--", -1)
-	return filepath.Join(o.root, "iscsiconfd", fmt.Sprintf("%s-%s.conf", name, id))
-}
-
-func (o *snapshotter) tgtTargetIqn(id string, key string) string {
-	name := strings.Replace(key, ":", "-", -1)
-	name = strings.Replace(name, "/", "--", -1)
-	return fmt.Sprintf("iqn.alibabacloud.overlaybd:%s.%s", name, id)
-}
-
-func (o *snapshotter) tgtTargetMountpoint(id string) string {
+func (o *snapshotter) overlaybdMountpoint(id string) string {
 	return filepath.Join(o.root, "snapshots", id, "block", "mountpoint")
 }
 
-func (o *snapshotter) tgtOverlayBDConfPath(id string) string {
+func (o *snapshotter) overlaybdConfPath(id string) string {
 	return filepath.Join(o.root, "snapshots", id, "block", "config.v1.json")
 }
 
-func (o *snapshotter) tgtOverlayBDInitDebuglogPath(id string) string {
+func (o *snapshotter) overlaybdLoopbackDeviceID(id string) string {
+	loopID := id
+	leadings := 13 - len(loopID)
+	for i := 0; i < leadings; i++ {
+		loopID = "0" + loopID
```

Comment:
you can use https://golang.org/pkg/strings/#Repeat instead loop here.

Reply:
Done